packages/askui-nodejs/src/execution/dsl.ts

```diff
@@ -12,6 +12,8 @@ export enum Separators {
   STRING = '<|string|>',
 }
 
+export type Relations = 'nearestTo' | 'leftOf' | 'above' | 'rightOf' | 'below' | 'contains';
```

Comment:
I think we should improve on the naming as this does not contain all relations but only a subset that are relevant for a specific use case.

Reply:
Not sure if `Anchor` is a better name for it, but we used it already in the docs and it makes kinda sense.

I am open to better suggestions.

Reply:
Nice term but I think it may be ambiguous and like you said, we already use the term `Relations` extensively. As this is a subset of all possible relations, what about a name like `RelationsForConvenienceMethods` (also clunky, maybe you can think of a better prefix but the general idea is to use `Relations...` prefix and specify the subset afterwards)?

Reply:
My first programming language was Java, so clunkiness is not a problem for me 😉 .

I like your thinking there and would go with your suggestion `RelationsForConvenienceMethods`. Better clunky and expressive than not clunky and ambiguous.